app/views/posts/index.html.erb

```diff
@@ -1 +1,29 @@
-<h1>Here are the list of all posts for a given user</h1>
+<% if @posts.blank? %>
+  <h2>There are no post for this user yet.</h2>
+<% else %>
+  <div class="flex flex-row py-5 gap-5">
+    <div class="border border-black w-1/6 h-20"></div>
+    <div class = "border rounded border-black flex flex-row p-3 justify-between w-5/6">
+      <h1><%= @user.name%></h1>
+      <p>Number of posts: <%= @user.posts_counter %></p>
+    </div>
+  </div>
+  <% @posts.each_with_index do |post, index| %>
+    <div>
+      <div class = "border rounded border-black flex flex-col p-2 gap-3">
+        <h3 class = "p-0 hover:bg-blue-700"><%= link_to "Post ##{index +1}", user_post_path(@user, post) %></h1>
+        <p><%= post.title %></p>
+        <p><%= post.text %></p>
+        <p> Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter %> </p>
+      </div>
+      <div class = "border rounded border-black flex flex-col p-2 gap-3">
+        <% post.comments.each do | comment| %>
+          <div >
```

Comment:
Kindly use the `recent_comments` method to render the 5 most recent comments for each post as per project requirements.